### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Review by fredj — APPROVED** [review]

File: jstools/merge.py [hunk 1]
```diff
@@ -246,11 +246,11 @@ def cat_run(self, outfile, sections, uncompressed=False, strip_deps=True, compre
         license = StringIO()
         seen = []
         for name in newfiles:
-            print >> catted, cat[name]
+            print(cat[name], file=catted)
```
Comment (pmauduit) [inline jstools/merge.py:249]: FYI this breaks geOrchestra builds (this syntax does not pass under python 2.7.x), related downstream issue: https://github.com/georchestra/georchestra/issues/1773

Comment (pgiraud) [reply jstools/merge.py:249]: You can add this:
```python
from __future__ import print_function
```

Comment (sbrunner) [reply jstools/merge.py:249]: Why not using Python 3?

**Comment by fvanderbiest** [comment]
This change and the related pypi update has probably broken our builds.

eg, when our dependencies were expressed as 
```js
/*
 * @include Ext.ux/widgets/colorpicker/ColorPicker.js
 * @include Ext.ux/widgets/colorpicker/ColorMenu.js
 * @include Ext.ux/widgets/colorpicker/ColorPickerField.js
 */
```
... these files where minified in the same order, allowing Ext.ux.ColorPicker.prototype to be defined in Ext.ux.form.ColorPickerField. This is not the case anymore, hence the [broken JS builds](https://github.com/georchestra/georchestra/issues/1779).

This is showcased in eg:
 * old build = http://www.geopicardie.fr/mapfishapp/build/mapfishapp.js
 * new build = https://dev.ppige-npdc.fr/mapfishapp/build/mapfishapp.js

**Pull request merged**